src/tokens/polygonTokens.json

```diff
@@ -20518,5 +20518,23 @@
                 "website": "https://peercoin.net/"
             }
         }
+    },
+    {
+        "chainId": 137,
+        "name": "Polkamarkets",
+        "symbol": "POLK",
+        "decimals": 18,
+        "address": "0x996F19d4b1cE6D5AD72CEaaa53152CEB1B187fD0",
+        "logoURI": "https://raw.githubusercontent.com/Polkamarkets/polkamarkets-web/main/public/badge.svg",
+        "tags": ["pos", "fx", "erc20", "swapable"],
```

Comment:
The token seems to be mapped over fx bridge. Please remove the "pos" tag.

Reply:
@py-zoid done 👍

Reply:
Sorry for not pointing earlier, but could you add "rootTunnelAddress" as "0x0cc2caed31490b546c741bd93dbba8ab387f7f2c". [Here](https://github.com/maticnetwork/polygon-token-list/blob/532277fe4f44040a7e548226274593b8b1b17b88/src/tokens/polygonTokens.json#L20458) is an example you can follow.

We'll add documentation for these custom cases as well into the readme file soon, looks like we missed it. Thank you.

Reply:
@py-zoid just added 👍